scripts/actions/mobskills/asthenic_fog.lua

```diff
@@ -14,7 +14,7 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.DROWN, 25, 3, 120))
+    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.DROWN, 15, 3, 60))
```

Comment:
```suggestion
    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.DROWN, 15, 0, 60))
```
Jimmy's sheet says the drown is static and doesn't tick down, so it should probably have a 0 tick just in case. Currently Drown has no onTickEffect but who knows if it will in the future

Reply:
I had thought that meant how many server ticks between damage instances, now I know. Changed it to a zero.

Reply:
yeah, REGEN_DOWN is handled on every tick in `void CStatusEffectContainer::TickRegen(timer::time_point tick)` It's completely agnostic of the effect which is nice